web/app/themes/dp-timber-theme/components/style.scss

```diff
@@ -1 +1,9 @@
-// Add global styling here.
\ No newline at end of file
+// Add global styling here.
+.sb-show-main.sb-main-padded {
+	padding: 0;
+}
+
+body {
```

Comment:
do we need globally or just for storybook.

Reply:
yes we need that globally